Integrations/Cylance_Protect_v2/Cylance_Protect_v2.yml

```diff
@@ -962,10 +961,13 @@ script:
     - contextPath: Cylance.Device.Deleted
       description: A boolean to check if the device was deleted.
       type: boolean
+    description: Delete one or more devices from an organization.
+    execution: true
```

Comment:
whats that?

Reply:
It yelled at me because the yml didn't have those, and when I downloaded it from my Demisto instance they did exist so copied them from there